tests/unit/disco/components/TestAddon.js

```diff
@@ -146,9 +151,7 @@ describe(__filename, () => {
         error: FATAL_INSTALL_ERROR,
       });
       const error = root.find('.notification.error');
-      expect(error.find('p').html()).toContain(
-        'An unexpected error occurred during installation.',
-      );
+      expect(error.find('p').html()).toContain(getMessage(FATAL_INSTALL_ERROR));
```

Comment:
These tests are now a duplication of what you added to `utils/test_addons.js`. I would like to see all of them deleted except for one or two which you can use as integration tests to make sure `disco/Addon.js` is calling `getErrorMessage()` and rendering the result (similar to what you added in `pages/TestAddon.js`).